build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,9 +48,9 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
         {
             if (CheckIfAllBuildsHavePublished())
             {
-                string targetContainer = $"{Channel}/Binaries/Latest/";
+                string targetContainer = $"{AzurePublisher.Product.Sdk}/{Channel}/";
                 string targetVersionFile = $"{targetContainer}{CliNuGetVersion}";
-                string semaphoreBlob = $"{Channel}/Binaries/publishSemaphore";
+                string semaphoreBlob = $"{targetContainer}/publishSemaphore";
                 AzurePublisherTool.CreateBlobIfNotExists(semaphoreBlob);
                 string leaseId = AzurePublisherTool.AcquireLeaseOnBlob(semaphoreBlob);
```

Comment:
We weren't doing it and we never saw a break until 2am on the evening of getting the release builds out. It's a rare situation but I think it's the right approach.

---
In reply to: [67969144](https://github.com/dotnet/cli/pull/3669#discussion_r67969144) [](ancestors = 67969144)